oci-artificial-intelligence/ai-speech/customizations/customizations.md

```diff
@@ -103,11 +103,162 @@ To view your newly created customization, click on your customization's name fro
     From either the customization details page or list customizations page, click <strong>delete</strong> to delete your customization
         ![delete customization](./images/delete-customization.png " ")
 
+# Task 4: Using the customizations API
```

Comment:
use two # for this so it creates a collapsible title like:
 ## Task 4: Using the customizations API